Packaging.props

```diff
@@ -0,0 +1,57 @@
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <PreReleaseLabel>alpha</PreReleaseLabel>
+    <PackageDescriptionFile>$(ProjectDir)src/pkg/descriptions.json</PackageDescriptionFile>
+    <PackageLicenseFile>$(ProjectDir)src/pkg/dotnet_library_license.txt</PackageLicenseFile>
+    <PackageThirdPartyNoticesFile>$(ProjectDir)src/pkg/ThirdPartyNotices.txt</PackageThirdPartyNoticesFile>
+    <ReleaseNotes>TODO</ReleaseNotes>
```

Comment:
@leecow can you create an FWLink (https://go.microsoft.com/fwlink/?LinkID=799421) for future CoreRT release notes